core/src/Borrower.sol

```diff
@@ -88,6 +91,23 @@ contract Borrower is IUniswapV3MintCallback {
                            MAIN ENTRY POINTS
     //////////////////////////////////////////////////////////////*/
 
+    function warn() external {
```

Comment:
Want to emit an event?
`event insolvencyWarning (address unhealthyBorrower, uint256 amount0ToRepay, uint256 amount1ToRepay)`

`emit insolvencyWarning()`

Reply:
Next PR :)